### DISCUSSION TITLE
Arquivo .dae na Web

### DISCUSSION THREAD

**Discussion by deppbrazil** [frontendbr/forum#735] [Perguntas e Respostas]
#### Queria pedir ajuda de vcs se é possível abrir um arquivo .dae no browser.... ####

Para quem não sabe um arquivo .dae é um arquivo em 3D exportado do [Sketchup](https://www.sketchup.com/pt-BR)

Nesta [doc ](http://www.hardware.com.br/tutoriais/3dweb/importando-collada.html)é possível utilizando as bibliotecas do flash.... 

> massss como o flash morreu ... uhuuu =)

 #### Queria saber se alguém ja conseguiu hoje em dia visualizar um arquivo desse no browser renderizado com JavaScript... ####

**Comment by phsantiago** [comment]
Acho que com JS não é possível mas você ainda pode usar o Flash...

**Comment by vitorebatista** [comment]
On dom, 24 de set de 2017 at 23:09 Pedro H. Santiago <
notifications@github.com> wrote:

> Acho que com JS não é possível mas você ainda pode usar o Flash...
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/frontendbr/forum/issues/735#issuecomment-331759734>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABoScWjh8gpVAmEzOX18YtvW6V9ya1Xeks5slwtvgaJpZM4PgqzD>
> .
>
-- 
Enviado do meu iPad.

**Comment by icaromh** [comment]
Botei `open sketchup in js` no google e abri esses 3 links

- https://github.com/mrdoob/three.js/wiki/Using-SketchUp-Models
- https://knowledge.safe.com/articles/648/create-threejs-from-sketchup.html 
- https://stackoverflow.com/a/16947400

Se der certo, pinga aí ;)

**Comment by felipefialho** [comment]
@deppbrazil rolou?

**Comment by deppbrazil** [comment]
@LFeh man desculpa a demora, mas como aqui no trampo é tudo pra ontem e sempre fazendo mais uns 4 projetos ao mesmo tempo ainda não testei tudo, assim que funcionar te dou um salve!
